### PR TITLE
Added the maintainers list

### PR DESCRIPTION
**Pull request by eveyonline**
Thanks to the maintainers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ on your project.
 
 ## Limitations
 
-This tool can be used in the following scenarii :
+This tool can be used in the following scenaro :
```
Comment (alexpozzi) [inline README.md:94]:
```suggestion
This tool can be used in the following scenario :
```

Comment (nm2107) [reply README.md:94]: Regarding [wikiperia](https://en.wiktionary.org/wiki/scenarii) : 

> However, the modern plural of the Italian word [scenario](https://en.wiktionary.org/wiki/scenario#Italian) is [scenari](https://en.wiktionary.org/wiki/scenari#Italian).

So the fix should be `scenari` IMHO (I originally meant to write the plural of scenario, hence the two i's, but it looks like I wrote it wrong).

Comment (alexpozzi) [reply README.md:94]: Oh I got it wrong, from the PR I didn't get there were multiple ones!

File: README.md [hunk 2]
```diff
@@ -124,5 +124,10 @@ and then use `cargo` to launch the tool :
...
 $ cargo run -- <ssc_options>
 ```
-
+## Contribution guide
 Also, see our [contributing guide](CONTRIBUTING.md).
+
+## Maintainers 
+[nm2107](https://github.com/nm2107)
+[neghmurken](https://github.com/neghmurken)
+[jaljo](https://github.com/jaljo)
```
Comment (hlegay) [inline README.md:133]: :thinking: 
For me you can keep only Nico in maintainers.

**Review by hlegay — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (nm2107) [reply README.md:133]: @neghmurken , @jaljo , do you accept to be listed as maintainers ?

I'll cherry pick the commit to rebase against master once I've got your answers.

Comment (neghmurken) [reply README.md:133]: Sorry, I'll politely decline. I have absolutely no knowledge of this project or of Rust whatsoever

Comment (nm2107) [reply README.md:133]: No problem, thank you for answering :)

@jaljo, what about you ? (no obligations there :) )

**Review by eveyonline — COMMENTED** [review]
Corrected your suggestions :) 
Thank you

File: README.md (hunk 1 above)
Comment (eveyonline) [reply README.md]: However, the modern plural of the Italian word scenario is scenari. (not in english)

**Review by hlegay — APPROVED** [review]

**Pull request merged**